zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/initialize/IdentitySetupInitializer.java

```diff
@@ -84,6 +84,9 @@ private IdentitySetupRecord buildSetupRecord() {
     PlatformDefaultEntities.setupDefaultTenant(setupRecord);
     PlatformDefaultEntities.setupDefaultRoles(setupRecord);
 
+    // TODO: remove after implementing actual functionality
+    LOG.debug("Configured authorizations: {}", initialization.getAuthorizations());
+
```

Comment:
🔧 I like the idea behind this but I'd not include it in the PR purely because we are backporting the cahnges into the v8.8 branch so they could be pushed out at any time and this log message could be misleading to users i.e. they could think this has some impact in the code but actually doesn't